admin/c2cgeoportal_admin/schemas/metadata.py

```diff
@@ -1,5 +1,4 @@
 import json
-from json.decoder import JSONDecodeError
```

Comment:
Note that it is possible to do : 
try:
    import JSONDecodeError
except:
    JSONDecodeError = ValueError

Reply:
@arnaud-morvan why this?